spec/requests/relationship_operations_spec.rb

```diff
@@ -291,29 +291,33 @@
     context 'authorized for remove_to_many_relationship' do
       context 'not limited by policy scopes' do
         before do
-          allow_operations('remove_to_many_relationship', [
-            [article, comments_to_remove.first, :comments],
-            [article, comments_to_remove.second, :comments]
-          ])
+          allow_operation('remove_to_many_relationship', article, kind_of(Array), "comments")
```

Comment:
Could we restructure this to test for the actual comments instead of just `kind_of(Array)`? The `kind_of(Array)` might be a bit too vague spec and could cause bugs to slip through as it doesn't care about the type of the objects inside the array itself.

Would something like this work?

```rb
allow_operation(
  'remove_to_many_relationship',
  article,
  [comments_to_remove.first, comments_to_remove.second],
  :comments
)
```

That would be the best. If the specs fail because of this due to some matchers not working correctly, we should fix the matchers instead of trying to work around them by using less specific assertions.

Also, as you can see, let's try to use the symbol version for the relationship name in the assertions (`:comments` instead of `"comments"`) or otherwise it might result in an inconsistent API with the other authorizers)